libs/langchain-community/src/document_loaders/web/cheerio.ts

```diff
@@ -114,7 +123,8 @@ export class CheerioWebBaseLoader
       this.webPath,
       this.caller,
       this.timeout,
-      this.textDecoder
+      this.textDecoder,
+      this.headers
```

Comment:
This is passing headers via the `options` field.

Reply:
My bad